style.css

```diff
@@ -0,0 +1,423 @@
+@font-face {
+  font-family: Fraunces-font;
+  src: url(./assets/Fraunces-VariableFont_SOFT\,WONK\,opsz\,wght.ttf);
+}
+
+@font-face {
+  font-family: Fraunces-font-bold;
+  src: url(./assets/static/Fraunces_72pt\,Soft-Bold.ttf);
+}
+
+footer {
+  color: gray;
+}
+
+.date {
+  color: white;
+}
+
+.container {
+  width: 100%;
+}
+
+.one {
+  background-color: #f72210;
+  width: 100%;
+}
+
+.btn-warning {
+  font-size: 6px;
+  border: 1px solid rgba(240, 173, 78, 0.75);
+  border-radius: 40px;
+  background: yellow;
+  color: black;
+  font-weight: bold;

```

Comment:
- [ ] You can use bootstrap classes for these selectors please look them up here https://getbootstrap.com/docs/4.0/getting-started/introduction/